js/product-details.js

```diff
@@ -0,0 +1,255 @@
+      specialistSmoll.innerHTML = key;
+      specialistSmoll.classList.add("details-notes--italic");
+      valueConteinerSmoll.appendChild(specialistSmoll);
+
+      let specialistBig = document.createElement("div");
+      specialistBig.innerHTML = key;
+      specialistBig.classList.add("details-notes--italic");
+      valueConteinerBig.appendChild(specialistBig);
+
+      if (Object.prototype.toString.call(drinkKey[key]) == "[object Array]") {
+        for (i = 0; i < drinkKey[key].length; i++) {
+          let textSmoll = document.createElement("div");
+          textSmoll.innerHTML = drinkKey[key][i];
+          textSmoll.classList.add("details-notes--margin");
+          valueConteinerSmoll.appendChild(textSmoll);
+          let textBig = document.createElement("div");
+          textBig.innerHTML = drinkKey[key][i];
+          textBig.classList.add("details-notes--margin");
+          valueConteinerBig.appendChild(textBig);
+        }
+      } else {
+        let feedbackSmoll = document.createElement("div");
+        feedbackSmoll.innerHTML = drinkKey[key];
+        feedbackSmoll.classList.add("details-notes--margin");
+        valueConteinerSmoll.appendChild(feedbackSmoll);
+
+        let feedbackBig = document.createElement("div");
+        feedbackBig.innerHTML = drinkKey[key];
+        feedbackBig.classList.add("details-notes--margin");
+        valueConteinerBig.appendChild(feedbackBig);
+      }
+    }
+    conteinerSmoll.appendChild(valueConteinerSmoll);
+    conteinerBig.appendChild(valueConteinerBig);
+  } else if (Object.prototype.toString.call(drink[key]) == "[object Array]") {
+    let valueConteinerSmoll = document.createElement("div");
+    valueConteinerSmoll.classList.add("product-details-value--smoll");
+    valueConteinerSmoll.classList.add("product-details-value--smoll--none");
+    valueConteinerSmoll.classList.add("product-details-value--smoll--" + key);
+    let valueConteinerBig = document.createElement("div");
+    valueConteinerBig.classList.add("product-details-value--big");
+    valueConteinerBig.classList.add("product-details-value--big--none");
+    valueConteinerBig.classList.add("product-details-value--big--" + key);
+    for (i = 0; i < drink[key].length; i++) {
+      let textSmoll = document.createElement("div");
+      textSmoll.innerHTML = drink[key][i];
+      textSmoll.classList.add("details-notes--margin");
+      valueConteinerSmoll.appendChild(textSmoll);
+      let textBig = document.createElement("div");
+      textBig.innerHTML = drink[key][i];
+      textBig.classList.add("details-notes--margin");
+      valueConteinerBig.appendChild(textBig);
+    }
+    conteinerSmoll.appendChild(valueConteinerSmoll);
+    conteinerBig.appendChild(valueConteinerBig);
+  } else {
+    let valueConteinerSmoll = document.createElement("div");
+    valueConteinerSmoll.innerHTML = drink[key];
+    valueConteinerSmoll.classList.add("product-details-value--smoll");
```

Comment:
seems like you're repeating yourself
it's best practice called DRY - Dont Repeat Yourself
if you see some code repeated event 2 times - make a several function with it.
event better to split this BIG function into a 5-10 smaller.
also, by Clen Code your function should be less than 20 strings (in common, not always)
try to optimize it
also, remember that function names should tell anyone what exactly they do
so i could not to read all your code and try to understand what classes mean